extensions/amp-sidebar/amp-sidebar.md

```diff
@@ -67,7 +66,8 @@ for content within the sidebar to be displayed on other parts of the main conten
     - `<amp-social-share>`
 - The max-height of the sidebar is 100vh, if the height exceeds 100vh then a vertical scrollbar appears. The default height is set to 100vh in CSS and is overridable in CSS.
 - The width of the sidebar can be set and adjusted using CSS (minimum width is 45px).
-- Touch zoom is disabled on the `amp-sidebar` and it's mask when the sidebar is open.
+- Touch zoom is disabled on the `amp-sidebar` and its mask when the sidebar is open.
```

Comment:
Add a comment that `z-index` stacking can cause problems showing the sidebar if this is not a direct child of the body because a parent (e.g. with `z-index: 1`) could cause it to stack below other things like headers (e.g. with `z-index: 2`).